app/models/activity.rb

```diff
@@ -3,6 +3,10 @@ class Activity < ApplicationRecord
   validates_presence_of :identifier
   validates_uniqueness_of :identifier
 
+  attribute :recipient_region, :string, default: "998"
```

Comment:
I think these defaults will work fine for fund activities but I wonder how we'll be able to support these being changed for programmes and projects, any ideas?

Reply:
Possibly a set of env variables? 

Or maybe a helper along the lines of ActivityHelper. Although if the values needed to change than env variables are probably more flexible?

Reply:
I hadn't thought of environment variables, how might that look?

I wonder if having some initializer helpers might help. Something like:

```
class Activity

def setup_hierarchy_defaults
  case hierachy.class.name
  when fund setup_fund_defaults
…
end

def setup_fund_defaults
  self.recipient_region = "998"
  …
end

def setup_programme_detaults
  self.recipient_region = "2"
  …
end
```

```
class Staff::ActivitiesController < Staff::BaseController
  include ActivityHelper
  def create 
    @activity.hierarchy = hierarchy
    @activity.set_up_hierachy_defaults
    @activity.save(validate: false)
…

Reply:
I like the initialiser helpers more, env variables would be clunky and I don't think these values are going to change often (maybe?)